src/seedu/addressbook/data/AddressBook.java

```diff
@@ -99,6 +99,14 @@ public void removePerson(ReadOnlyPerson toRemove) throws PersonNotFoundException
         allPersons.remove(toRemove);
     }
 
+    /**
+     * Edits the equivalent person's name from the address book.
```

Comment:
The header comment does not follow the convention. All non trivial methods should have javadoc format header comments. (First line is the description of method, followed by @param for specifying the parameters and @return for specifying the return values, if applicable)

Reply:
In this method, you need to specify the exception @throw also.